llrt_core/src/modules/http/url.rs

```diff
@@ -238,6 +238,13 @@ impl<'js> URL<'js> {
         self.password.clone_from(&password);
         password
     }
+
+    #[qjs(rename = PredefinedAtom::ToJSON)]
+    fn to_json(&self) -> String {
+        // NOTE: It is not wrong to return as a string.
```

Comment:
I think we can remove this comment!

Reply:
Do you want me to delete the next comment as well?
```rust
// https://developer.mozilla.org/en-US/docs/Web/API/URL/toJSON
```

Reply:
Thats fine to keep!